scripts/lib/CIME/preview_namelists.py

```diff
@@ -94,9 +95,10 @@ def create_namelists(case):
             raise
 
         if do_run_cmd:
-            logger.debug("   Running %s buildnml"%compname)
+            logger.info("   Running %s buildnml"%compname)
```

Comment:
I think I changed this because in the old implementation, all the output from buildnml was completely lost and I had trouble finding something.